rollup.config.js

```diff
@@ -0,0 +1,6 @@
+const config = {
+  input: 'src/custom-elements.js',
+  output: { exports: 'named', file: 'custom-elements.min.js', format: 'iife', name: 'CustomElements', sourcemap: true }
```

Comment:
Could you remove the `exports: 'named',` and `name: 'CustomElements',` options here? We don't actually want things from the polyfill to show up anywhere except through the API it fakes (`window.customElements`, etc.). I tried running both with and without those options and it produced the same output on my machine, so I'm guessing they only apply when using some format that actually has named exports (i.e. not `iife`)? Either way, I think it's safer to remove these, unless there's another reason that I haven't thought of.